resolwe_bio/processes/samtools/samtools_coverage.py

```diff
@@ -0,0 +1,162 @@
+"""Samtools coverage."""
+
+from plumbum import TEE
+
+from resolwe.process import (
+    BooleanField,
+    Cmd,
+    DataField,
+    FileField,
+    GroupField,
+    IntegerField,
+    ListField,
+    Process,
+    SchedulingClass,
+    StringField,
+)
+
+
+class SamtoolsCoverage(Process):
+    """Samtools coverage.
+
+    Computes the depth at each position or region and creates tabulated text.
+
+    For more information about samtools coverage, click
+    [here](https://www.htslib.org/doc/samtools-coverage.html).
+    """
+
+    slug = "samtools-coverage"
```

Comment:
Would a process operating on a single BAM input ( with `entity = {"type": "sample"}`) be useful in addition to the one that operates on multiple BAMs?

Reply:
Great suggestion